includes/tax-functions.php

```diff
@@ -442,3 +410,62 @@ function edd_download_is_tax_exclusive( $download_id = 0 ) {
 
 	return (Bool) apply_filters( 'edd_download_is_tax_exclusive', $ret, $download_id );
 }
+
+/**
+ * Gets the tax rate object from the database for a given country / region.
+ * Used in `edd_get_tax_rate`, `edd_build_order`, `edd_add_manual_order`.
+ * If a regional tax rate is found, it will be returned immediately,
+ * so rates with a scope of `country` may be overridden by a more specific rate.
+ *
+ * @param array $args {
+ *     Country and, optionally, region to get the tax rate for.
+ *
+ *     @type string $country Required - country to check.
+ *     @type string $region  Optional - check a specific region within the country.
+ * }
+ * @return object|boolean
```

Comment:
One minor change:

Instead of `@return object` I'd like to actually specify the exact object. That led me to a discovery that I'm not sure if there's a reason for...

The `edd_get_tax_rates()` function specifies that if `$output = OBJECT` then it will return `an array of EDD\Adjustments\Adjustment objects`. But that's not actually correct. In `EDD\Database\Queries\Adjustment`, the `$item_shape` is set to `\\EDD\\Database\\Rows\\Adjustment`.

I guess for now, for the scope of this PR, we can specify what it actually currently is, but maybe it's worth making a new issue to look into this inconsistency? Whether that means changing the `$item_shape` to be `\\EDD\\Adjustments\\Adjustment` or changing the comment in `edd_get_tax_rates()`.

But for now...

```suggestion
 * @return \EDD\Database\Rows\Adjustment|false
```